schema/xcodegen.schema.json

```diff
@@ -0,0 +1,1390 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
`m`: How do we keep this up to date? Does it make sense to generate this automatically in CI or should we add a makefile command? Or how does this work?